kafka/api.html

```diff
@@ -2278,11 +2298,20 @@ <h3 id="Consumer.Assign">
     <pre>func (c *<a href="#Consumer">Consumer</a>) Assign(partitions []<a href="#TopicPartition">TopicPartition</a>) (err <a href="//golang.org/pkg/builtin/#error">error</a>)</pre>
     <p>
      Assign an atomic set of partitions to consume.
-This replaces the current assignment.
+    </p>
+    <p>
+     The .Offset field of each TopicPartition must either be set to an absolute
+starting offset (&gt;= 0), or one of the logical offsets (`kafka.OffsetEnd` etc),
+but should typically be set to `kafka.OffsetStored` to have the consumer
```

Comment:
In .net I use -1001 for this. it would have been better if that were not allowed..